<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>175</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">Rufus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rufusresource.qrc">
    <normaloff>:/res/icon-rufus.svg</normaloff>:/res/icon-rufus.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="titleHorizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="controlsGridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="imageEdit">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="deviceLabel">
       <property name="text">
        <string>&amp;USB Device:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>deviceList</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="imageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Image:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="deviceList">
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="imageSelectButton">
       <property name="toolTip">
        <string>Open image file</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="rufusresource.qrc">
         <normaloff>:/res/icon-open.ico</normaloff>:/res/icon-open.ico</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="deviceRefreshButton">
       <property name="toolTip">
        <string>Refresh the list</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="rufusresource.qrc">
         <normaloff>:/res/icon-refresh.ico</normaloff>:/res/icon-refresh.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%v / %m MB</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="progressBarSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Delete partitions and make all space available for storing data</string>
       </property>
       <property name="text">
        <string>Clea&amp;r</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="writeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>imageEdit</tabstop>
  <tabstop>imageSelectButton</tabstop>
  <tabstop>deviceList</tabstop>
 </tabstops>
 <resources>
  <include location="rufusresource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>imageSelectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>openImageFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>writeImageToDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceRefreshButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>scheduleEnumFlashDevices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>clearDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImageFile()</slot>
  <slot>writeImageToDevice()</slot>
  <slot>scheduleEnumFlashDevices()</slot>
  <slot>clearDevice()</slot>
 </slots>
</ui>
